receiver/githubactionsreceiver/documentation.md

```diff
@@ -29,3 +29,21 @@ Number of jobs.
 | ci.github.workflow.job.status | Job status | Str: ``completed``, ``in_progress``, ``queued``, ``waiting``, ``aborted`` |
 | ci.github.workflow.job.conclusion | Job Conclusion | Str: ``success``, ``failure``, ``cancelled``, ``neutral``, ``null``, ``skipped``, ``timed_out``, ``action_required`` |
 | ci.github.workflow.job.head_branch.is_main | Whether the head branch is the main branch | Any Bool |
+
+### workflow.runs.count
```

Comment:
That's across all workflows of a specific repository, right? Would it perhaps also make sense to have another metric (or an attribute for this) for getting the number of runs for a specific workflow name inside a project?